package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@teamsnap/teamsnap-ui",
-  "version": "4.0.0-alpha.95",
```

Comment:
While you're free to do this, this does not need to be updated anymore. It's processed from the git tag by semantic release before publishing. If we want to have it automatically update the version, thats something I can look at, but manually updating this is no longer needed :)

Reply:
That's terrific news. Thanks!